scripts/docker/entrypoint_ci.sh

```diff
@@ -221,8 +221,12 @@ function check_boto_upgrade() {
     echo
     echo "${COLOR_BLUE}Upgrading boto3, botocore to latest version to run Amazon tests with them${COLOR_RESET}"
     echo
-    pip uninstall --root-user-action ignore aiobotocore s3fs -y || true
-    pip install --root-user-action ignore --upgrade boto3 botocore
+    # shellcheck disable=SC2086
+    ${PACKAGING_TOOL} uninstall ${EXTRA_UNINSTALL_FLAGS} aiobotocore s3fs || true
```

Comment:
Do we not need a `-y` for these commands?

Reply:
They are added in "EXTRA_UNINSTALL_FLAGS" for `pip`(I aded it there as full long option `--yes` ).  The `uv pip uninstall` does not take the `-y` or `--yes` flag so we cannot add it in this case (and EXTRA_UNINSTALL_FLAGS for `pip uv` are empty because of that.

Reply:
I see. Thank you. Missed it